File: php/02-public-vs-private/public-vs-private.php
<?php
class Person
{
    public $name;
    public $age;

    // set default value for id
    private $id = "1";

    public function printData()
    {
        echo "Name: " . $this->name . "\n";
        echo "Age: " . $this->age . "\n";
        echo "Id: " . $this->id . "\n";
    }
}

// create object
$personObject = new Person();

// set properties
$personObject->name = "Seyed Reza Bazyar";
$personObject->age = 25;
// can't access privates from outside of the class
# $personObject->id = "new_id"; //Error

$personObject->printData();

File: cpp/02-public-vs-private/public-vs-private.cpp
#include <string>
#include <iostream>

using namespace std;

class Person
{
private:
    // set default value for id
    string id = "1";
    
public:
    string name;
    int age;

    void printData()
    {
        cout << "Name: " << name << endl;
        cout << "Age: " << age << endl;
        cout << "Id: " << id << endl;
    }
};

int main()
{
    // create object
    Person personObject;

    // set properties
    personObject.name = "Seyed Reza Bazyar";
    personObject.age = 25;
    // can't access privates from outside of the class
    // personObject.id = "new_id"; //Error

    personObject.printData();
}